--- nginx.spec.orig     2017-01-31 15:37:16.000000000 +0000
+++ nginx.spec  2017-03-20 10:20:57.500197865 +0000
@@ -78,6 +78,7 @@
 Source11: nginx-debug.service
 Source12: COPYRIGHT
 Source13: nginx.check-reload.sh
+Source14: headers-more-nginx-module-0.32.tar.gz

 License: 2-clause BSD-like license

@@ -97,6 +98,7 @@

 %prep
 %setup -q
+%setup -b 14 -q
 cp %{SOURCE2} .
 sed -e 's|%%DEFAULTSTART%%|2 3 4 5|g' -e 's|%%DEFAULTSTOP%%|0 1 6|g' \
     -e 's|%%PROVIDES%%|nginx|g' < %{SOURCE2} > nginx.init
@@ -107,13 +109,15 @@
 ./configure %{BASE_CONFIGURE_ARGS} \
     --with-cc-opt="%{WITH_CC_OPT}" \
     --with-ld-opt="%{WITH_LD_OPT}" \
+    --add-module=../headers-more-nginx-module-0.32 \
     --with-debug
 make %{?_smp_mflags}
 %{__mv} %{bdir}/objs/nginx \
     %{bdir}/objs/nginx-debug
 ./configure %{BASE_CONFIGURE_ARGS} \
     --with-cc-opt="%{WITH_CC_OPT}" \
-    --with-ld-opt="%{WITH_LD_OPT}"
+    --with-ld-opt="%{WITH_LD_OPT}" \
+    --add-module=../headers-more-nginx-module-0.32
 make %{?_smp_mflags}

 %install
